docs/int_header/INT_header.md

```diff
@@ -9,83 +9,61 @@ These definitions contain modifications and required fields for Transparent Secu
 This header heavily leverages the P4 INT header.  The changes and subset that are required for
 Transparent Security noted in this document.
 
-The current draft of the 2.0 INT header document is located at [INT.pdf](https://github.com/p4lang/p4-applications/blob/master/docs/INT.pdf)
+The draft of the 2.0 INT header document which we are using is located at [INT.pdf](https://github.com/cablelabs/transparent-security/wiki/attachments/INT.pdf).
 
 ## Overview
 
-The INT IP shim header and the INT header(s) will be inserted after the IP header and before the datagram for IPv4 and will be a part of the extended header for IPv6.  The IP header will be updated to indicate that it has an INT header.
+The INT header is defined in two portions.  One is the Header for the INT metadata and the second is the actual metadata.  For both of these, we follow the INT specification with a domain specific extension for the MAC address of the source device.
 
-The INT header is defined in two portions.  One is the Header for the INT metadata and the second is the actual metadata.  For both of these, we follow the INT specification with the addition of two bitmasks.  This additional value, an 8 octet value for the originating device ID will be proposed to be added to the INT specification.
+Please refer to the 2.0 draft or release version of the P4 Application for the metadata header format.  This document includes the definition of the domain specific extension and examples using a TCP/UDP header to encapsualte the packet.
```

Comment:
Do you mean "example using UDP header to encapsulate the packet"?  The spec doesn't offer an option to encapsulate with TCP...